unit/java_bytecode/generate_concrete_generic_type/generate_java_generic_type.cpp

```diff
@@ -184,8 +184,9 @@ SCENARIO(
 
   // We want to test that the specialized/instantiated class has it's field
   // type updated, so find the specialized class, not the generic class.
-  const irep_idt test_class=
-    "java::generic_field_array_instantiation$generic<java::array[reference]>";
+  const irep_idt test_class =
+    "java::generic_field_array_instantiation$generic<java::array[reference]of_"
```

Comment:
Very minor nit: could you break this string after the `]`, in the same way that you've broken it for `test_class_integer` below, as that seems more readable to me.

Reply:
ok, but aren't we using `clang-format` to not do this manually anymore?

Reply:
I don't have a strong feeling either way, so feel free to ignore me :-) I just felt the extra break made it more readable. `clang-format` gets around all the 'no space after =` stuff I guess - but its not clear to me how much we're meant to slavishly follow `clang-format`